strr-platform-web/app/locales/en-CA.ts

```diff
@@ -1,12 +1,24 @@
 /* eslint-disable max-len */
 export default {
+  act: {
+    foippa: 'Freedom of Information and Protection of Privacy Act'
```

Comment:
one of my 'isms
it's FIPPA, the OCIO calls it FOIPPA but they don't own that part of the legislation, the OIPC does and they like most other agencies in the world call it FIPPA